setup.py

```diff
@@ -1,46 +1,58 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
 
-with open('README.rst') as file:
-    long_description = file.read()
+# read the contents of your README file
+from os import path
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.rst'), encoding='utf-8') as f:
+    long_description = f.read()
 
-setup(
-        name = "sifter",
-        version = "0.1",
-        author = "Gary Peck",
-        author_email = "gary@realify.com",
-        url = "https://github.com/garyp/sifter",
-        license = "BSD",
-        description = "Parser/evaluator for the Sieve filtering language (RFC 5228)",
-        long_description = long_description,
-        classifiers = [
-            "Programming Language :: Python",
-            "Programming Language :: Python :: 2",
-            "License :: OSI Approved :: BSD License",
-            "Development Status :: 4 - Beta",
-            "Intended Audience :: Developers",
-            "Intended Audience :: System Administrators",
-            "Operating System :: OS Independent",
-            "Topic :: Communications :: Email :: Filters",
-            "Topic :: Software Development :: Interpreters",
-            "Topic :: Software Development :: Libraries :: Python Modules",
-            ],
-        packages = [
-            "sifter",
-            "sifter.commands",
-            "sifter.comparators",
-            "sifter.extensions",
-            "sifter.grammar",
-            "sifter.t",
-            "sifter.tests",
-            "sifter.validators",
-            ],
-        package_data = {
-            "sifter.t" : ["*.in", "*.out", "*.msg", "*.rules"],
-            },
-        install_requires=[
-            "ply"
```

Comment:
Don't we still need the dependency to be in here? I haven't been using Python as much recently, so I'm not sure if conventions have changed. But usually projects put their dependencies into `install_requires` so that they'll be properly included in the pip package info.

Reply:
Those files are only needed by unit tests. unit tests are not included in the package. So there is no need to include the files.

We can include unit tests in the source distribution, but I would recommend to keep the releases free from development stuff.

Reply:
I agree with you on the files related to unit tests. But that's not what I was commenting on. I was asking whether `install_requires=["ply"]` should still be included?